### PR TITLE
Create Asia Model and Widget. Use everywhere.

### LINKED ISSUE #212: Instrument: Implement Standard ASIA Widget and use in all pages.
The following sections need ASIA Widget:

Patient/SCI/D Information 
Annual Evaluation/Asia and NLI 
Acute Rehab/Start 
Acute Rehab/Finish 
OMR/Start 
OMR/Finish

### PR DESCRIPTION
**Pull request by awong-dev**
Fixes #212 minimally.

### CODE REVIEW

**Comment by awong-dev** [comment]
Hey @emilyville.  Wanna do a HUGE review?  Of course you do. kthxbai.

File: app/views/shared/_asia_fields.html.slim [hunk 1]
```diff
@@ -17,22 +17,22 @@
...
+                  = f.input_field :neurological_motor_level_right, label: false, collection: Asia.level_of_injuries.keys
       .col-md-4
         label ASIA Impairment Scale
-        = f.input :asia_score, :prompt => "Select one", collection: Patient.asia_levels.keys, :label => false, input_html: { class: "form-control" }
+        .form-control.input Missing JS auto-calc
         hr
 
         label Neurological Level of Injury
-        = f.input :level_of_injury, :label => "Neurological Level of Injury", :prompt => "Select one", collection: Patient.level_of_injuries.keys, :label => false, :no_col => true
+        .form-control.input Missing JS auto-calc
```
Comment (emilyville) [inline app/views/shared/_asia_fields.html.slim]: Is "Missing JS auto-calc"  a note for later or left in error?

**Comment by emilyville** [comment]
Looks good, and loaded locally and everything works fine. 
It does not display previously entered scores elsewhere. (i.e. if we I enter assessment on edit patient screen I only see it there and not on the annual assessment screen) but this could be a bug we file for later given the current timeline... 

If that's the case, happy to merge now!

File: app/views/shared/_asia_fields.html.slim (hunk 1 above)
Comment (awong-dev) [inline app/views/shared/_asia_fields.html.slim]: Yep...note for later.  The Asia Classification is an algorihm based off the other scores. You shouldn't have to type it in.

The level of injury is basically find the "highest" value of given 8 measurements.

**Pull request merged**